homeassistant/components/onkyo/media_player.py

```diff
@@ -236,8 +314,18 @@ def __init__(
         self._source_list = list(sources.values())
         self._source_mapping = sources
         self._reverse_mapping = {value: key for key, value in sources.items()}
+        self._sound_mode = None
+        self._sound_mode_list = list(SOUND_MODE_MAPPING.keys())
+        self._sound_mode_mapping = SOUND_MODE_MAPPING
```

Comment:
Why store this on the instance instead of just referencing constant directly?

Reply:
I tried to keep the same syntax as for the source mapping: `self._source_mapping = sources`
The difference is of course that, unlike sound modes, sources may be set in the configuration

Reply:
Exactly, so no need to make it an instance variable :)